src/pages/signup-affiliates/components/_phone_number.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import OtpVerification from 'images/svg/signup-affiliates/code 2.svg'
+import { Input, Button } from 'components/form'
+
+const PhoneNumber = () => {
+    const [phone, setPhone] = useState('')
+    const [phonecode, setPhoneCode] = useState('44')
+    const handleInputNameChange = (e) => {
```

Comment:
Looks like we are changing the phone value here but the name of the function is `handleInputNameChange`

Reply:
okay changed